WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Customs/WooShippingCustomsItemViewModel.swift

```diff
@@ -0,0 +1,33 @@
+import Yosemite
+import SwiftUI
+
+final class WooShippingCustomsItemViewModel: ObservableObject {
+    @Published var title: String
+    @Published var description: String
+    @Published var hsTariffNumber: String
+    @Published var valuePerUnit: String
+    @Published var weightPerUnit: String
+    @Published var originCountry: Country
+
+    var informationIsMissing: Bool = true
+
+    let allCountries: [Country]
+
+    let hsTariffURL: URL? = .init(string: "https://woocommerce.com/document/woocommerce-shipping-and-tax/woocommerce-shipping/#section-29")
```

Comment:
Should we add this to `WooConstants.URLs` since it's a known/trusted URL?

Reply:
Indeed, it looks much better there 00c7eb6c9863b957c5a4709f4694b626f678a4d4